gensim/models/word2vec.py

```diff
@@ -1799,8 +1799,9 @@ def predict_output_word(self, context_words_list, topn=10):
 
         Parameters
         ----------
-        context_words_list : list of str
-            List of context words.
+        context_words_list : list of str OR list of int
+            If list of str: List of context words.
+            If list of int: List of indices of context words in `self.wv.vectors`
```

Comment:
This comment implies the list must be all one or the other - when in fact the code supports a mix of `str` & `int`.

Reply:
Perhaps instead:

```
    context_words_list : list of (str or int)
        List of context words, each by string key or int index.
```

Reply:
Oops I forgot to reread this, you're right.